presto-product-tests/bin/run_on_docker.sh

```diff
@@ -83,6 +83,8 @@ function stop_application_runner_containers() {
     docker stop ${CONTAINER_NAME}
     echo "Container stopped: ${CONTAINER_NAME}"
   done
+  echo "Removing dead application-runner containers"
+  docker ps -aq --no-trunc --filter status=dead --filter status=exited | xargs docker rm -v || true
```

Comment:
Since the application runner containers were started using docker run( not `docker-compose`), I don't think we can remove using compose, but I'll check.
Regarding `removing much more` do you mean volumes.